recipe/meta.yaml

```diff
@@ -10,48 +10,41 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
   skip: true  # [py2k]
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - python
-    - setuptools
+    - cyipopt >=0.3
+    - cython >=0.24
     - matplotlib-base
-    - pandas
-    - nose
-    - mock
-    - xarray >=0.11.2
-    - sympy ==1.5.1
-    - pyparsing
-    - tinydb >=3.8
-    - scipy
     - numpy >=1.13
-    - dill
-    - cython >=0.24
-    - cyipopt
+    - pyparsing
+    - python
     - python-symengine ==0.6.1
+    - scipy
+    - setuptools
     - symengine ==0.6.0
+    - sympy ==1.5.1
+    - tinydb >=3.8
+    - xarray >=0.11.2
 
   run:
-    - python
+    - cyipopt >=0.3
+    - cython >=0.24
```

Comment:
Is Cython a runtime dependency of pycalphad?

Reply:
I think technically no, but it is nice to have for symmetry in the requirements (since we don't do build/host/run differentiation elsewhere) and it helps to eliminate a step if a user wants to install a development version